rwth_upper_body_skeleton_random_walk/launch/fixed.launch

```diff
@@ -0,0 +1,20 @@
+<launch>
+
+<include file ="$(find ground_plane_estimation)/launch/ground_plane_fixed.launch"/>
+<include file ="$(find upper_body_detector)/launch/upper_body_detector.launch"/>
```

Comment:
Rectified the version number, author name, license , launch file and other
issues mentioned by chris.

On Tue, Feb 16, 2016 at 11:29 AM, Christian Dondrup <
notifications@github.com> wrote:

> In rwth_upper_body_skeleton_random_walk/launch/fixed.launch
> <https://github.com/strands-project/strands_perception_people/pull/178#discussion_r52992761>
> :
>
> > @@ -0,0 +1,20 @@
> > +<launch>
> > +
> > +<include file ="$(find ground_plane_estimation)/launch/ground_plane_fixed.launch"/>
> > +<include file ="$(find upper_body_detector)/launch/upper_body_detector.launch"/>
>
> You can also add it to the *_robot.launch file in that packed directly.
> Maybe with a parameter so one can choose to start the people perception
> with or without it. In any case, please create the minimal launch file Marc
> asked for, then I can include it into the overall system launch file
> afterwards.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/strands-project/strands_perception_people/pull/178/files#r52992761>
> .
>